src/args.rs

```diff
@@ -6,25 +6,24 @@ use clap::{command, Parser};
     version, 
     about, 
     long_about = r###"
-Queries can be piped in through stdin and will be added to the context
```

Comment:
No pressure on this, I can change it back.. just got started writing the other for config and added it when thinking about the --help layout